javascript/javascript3/week2/js3week2.js

```diff
@@ -0,0 +1,104 @@
+// Movies exercise
+
+const badAndOld = fetch(
+    "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+  )
+    .then((response) => response.json())
+    .then(data => { 
+        data
+        .filter(movie => movie.rating <= 5)
+        .filter(movie => movie.year >= 2000);
+      console.log(data);
+    })
+    .catch((error) => console.log(error));
+    console.log(badAndOld);
+
+    // Promise that resolves after set time
+
+      
+
+    function newFunc(resolveAfter) {
+        return new Promise(resolve => setTimeout(resolve, resolveAfter));
+    }
+
+      newFunc(5000).then(() => console.log("Done"));
+
+    async  function newFunc(resolveAfter) {
+        await new Promise(resolve => setTimeout(resolve, resolveAfter));
+    }
+
+      newFunc(5000).then(() => console.log("Done"));
```

Comment:
you call the async function with the await, that keyword will ask the execution to wait until the defined task gets executed